.gitignore

```diff
@@ -1,2 +1,18 @@
-/target
-.DS_Store
```

Comment:
Can we keep .DS_Store? 

It is a default file added on MacOS.

Reply:
Yes, keep it.